README.md

```diff
@@ -180,6 +180,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 ### Database
 
 * [indradb](https://crates.io/crates/indradb) — Rust based graph database [<img src="https://api.travis-ci.org/indradb/indradb.svg?branch=master">](https://travis-ci.org/indradb/indradb)
+* [Materialize](https://github.com/MaterializeInc/materialize) - Streaming SQL database powered by Timely Dataflow [![Build status](https://badge.buildkite.com/97d6604e015bf633d1c2a12d166bb46f3b43a927d3952c999a.svg?branch=main)](https://buildkite.com/materialize/tests)
```

Comment:
As this is a commercial product, please add the `:heavy_dollar_sign:` (:heavy_dollar_sign:) emoji after the name in teh description.

Reply:
done!